### PR TITLE
Doc: update integrated-example doc

### PR DESCRIPTION
**Pull request by yuluo-yx**
update integreated-exmaple document!

### CODE REVIEW

File: spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md [hunk 1]
```diff
@@ -80,11 +73,9 @@ docker-compose-env.yml文件运行成功之后，添加Nacos配置：
 - 扣减账户余额
 - 生成订单信息返回响应
 
-##### 启动测试
+**开始测试**
```
Comment (steverao) [inline spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md]: Please check whether is a wrong here? use `#### 启动测试`? Here isn't consistent with English version

File: spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md [hunk 2]
```diff
@@ -117,27 +108,25 @@ docker-compose-env.yml文件运行成功之后，添加Nacos配置：
 - Sentinel 在网关侧绑定指定网关路由进行服务的熔断降级。
 - RocketMQ 进行流量削峰填谷，在大流量请求下，生产者向 RocketMQ 发送消息，而消费者则通过可配置的消费速率进行拉取消费，减少大流量直接请求数据库增加点赞请求的压力。
 
-#### 启动测试
-
-分别启动`integrated-praise-provider`以及`integrated-praise-consumer`模块。
+**开始测试**
```
Comment (steverao) [inline spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md]: Same problem.

File: spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md (hunk 1 above)
Comment (yuluo-yx) [reply spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md]: > Please check whether is a wrong here? use `#### 启动测试`? Here isn't consistent with English version

Done!

File: spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md (hunk 2 above)
Comment (yuluo-yx) [reply spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md]: > Same problem.

Done!

File: spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ docker-compose-env.yml文件运行成功之后，添加Nacos配置：
 - 扣减账户余额
 - 生成订单信息返回响应
 
-**开始测试**
+**启动测试**
```
Comment (steverao) [inline spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md]: 格式还是不一致，英文版本是#### XXX，中文版本为什么变成了 **XXX**?

Comment (yuluo-yx) [reply spring-cloud-alibaba-examples/integrated-example/docs/zh/docker-compose-deploy-zh.md]: > 格式还是不一致，英文版本是#### XXX，中文版本为什么变成了 **XXX**?

Sorry, I thought it was a problem with the text not corresponding, it has been changed!

**Review by steverao — APPROVED** [review]
LGTM, thanks @yuluo-yx

**Pull request merged**